src/Momento.Sdk/Internal/ScsTopicClient.cs

```diff
@@ -198,7 +198,7 @@ public async Task Subscribe()
             _subscribed = true;
         }
 
-        public async ValueTask<TopicMessage?> GetNextRelevantMessageFromGrpcStreamAsync(
+        public async ValueTask<ITopicEvent?> GetNextRelevantMessageFromGrpcStreamAsync(
```

Comment:
If we change this to return system events as well, we could shorten my Java-length method name choice here.

Reply:
Done in https://github.com/momentohq/client-sdk-dotnet/pull/575/commits/9d8a4bb99e29aee72b0806c829ffb2818d722520